drivers/staging/vc04_services/bcm2835-audio/bcm2835.c

```diff
@@ -106,17 +110,30 @@ static struct bcm2835_audio_driver bcm2835_audio_alsa = {
 	.newctl = snd_bcm2835_new_ctl,
 };
 
-static struct bcm2835_audio_driver bcm2835_audio_hdmi = {
+static struct bcm2835_audio_driver bcm2835_audio_hdmi0 = {
 	.driver = {
 		.name = "bcm2835_hdmi",
 		.owner = THIS_MODULE,
 	},
-	.shortname = "bcm2835 HDMI",
-	.longname  = "bcm2835 HDMI",
+	.shortname = "bcm2835 HDMI 0",
```

Comment:
Ditto this change.

Reply:
Should I simply rename this back, and assume its zero, leaving 1 as is?

Reply:
That sounds reasonable.

Reply:
Changed back